src/client/vue/pages/PagePlayRemote.vue

```diff
@@ -245,6 +245,24 @@ const initGameView = async () => {
     }
 };
 
+const makeTitle = (gameClient: HostedGameClient) => {
+    const players = gameClient.getPlayers();
+    const { state } = gameClient.getHostedGame();
+    if (players.length < 2 && 'created' === state)
+        return `Waiting for an opponent: hosted by ${players[0].pseudo}`;
+    const pairing = players.map(p => p.pseudo).join(' VS ');
+    const capitalizedState = state[0].toUpperCase() + state.slice(1);
+    let yourTurn = '';
+    if (loggedInPlayer.value != null) {
+        const player = loggedInPlayer.value;
+        const index = players.findIndex(p => p.publicId === player.publicId);
+        if (index != null && gameClient.getGame().getCurrentPlayerIndex() === index) {
```

Comment:
Should be added only if game is still playing, to prevent "your turn" on ended/canceled games

```suggestion
        if (index != null && 'playing' === gameClient.getState() && gameClient.getGame().getCurrentPlayerIndex() === index) {
```

Reply:
Right, good catch